src/UI/class-admin-bar.php

```diff
@@ -73,7 +73,7 @@ public function admin_bar_parsely_stats_button( WP_Admin_Bar $admin_bar ): void
 		 */
 		$current_object = $GLOBALS['wp_the_query']->get_queried_object();
 
-		if ( null === $current_object || '' === $current_object->post_type ) {
+		if ( null === $current_object || null === $current_object->post_type || '' === $current_object->post_type ) {
```

Comment:
We were continuing even if `$current_object->post_type` was null, which was creating warnings.